app/models/manageiq/providers/google/cloud_manager.rb

```diff
@@ -54,6 +61,21 @@ def verify_credentials(auth_type = nil, _options = {})
   #
 
   def connect(_options = {})
+    require 'fog/google'
+
+    raise MiqException::MiqHostError, "No credentials defined" if self.missing_credentials?(:service_account)
+
+    service_account = authentication_service_account(:service_account)
+
+    options = {
+      :provider               => "Google",
+      :google_project         => project,
+      :google_json_key_string => service_account,
+    }
+
+    connection = ::Fog::Compute.new(options)
```

Comment:
I originally pulled this from the verify_credentials where other checks were being done after the connect but it can be removed now.